### ISSUE TITLE
Docs: Create a device/wallet guide

### ISSUE THREAD

**Issue by k9ert** [cryptoadvance/specter-desktop#1885]
We're completely lacking documentation about the most basic things to do in specter. This issue is fixed with a document on how to create a device and based on that, how to create a wallet. So eventually this should be two entries in the `User Guide` of the [Specter Desktop Documentation](https://docs.specter.solutions/desktop).

The device-creation should explain different types of hardware-wallets (SD-Card/QR/USB) and the challenges about derivation-pathes. In general the document should explain each click and should go in detail about the choices. Links to external websites going in more detail should be included (e.g. https://learnmeabitcoin.com/technical/derivation-paths ).

The wallet-creation document should also briefly explain the differences of user-choices but mainly focus on single-sig. Later articles can focus on multi-sig. Included should be an explanation of the (different) backup-pdfs and what they basically contain (not the seed!).

Some resources which might help here:
* [25 mins video introduction to Markdown in Visual Studio Code with Markdown worksheet](https://www.youtube.com/watch?v=pTCROLZLhDM), a good editor to deal with markdown
* [7 mins Video Using Git with Visual Studio Code (Official Beginner Tutorial)](https://www.youtube.com/watch?v=i_23KUAEtUM)
* [5 mins tutorial howto make changes to projects in github with PRs (and forks)](https://www.youtube.com/watch?v=4ZlK55_44II)

This issue is part of the [Nodesignal Bounty Program](https://nodesignal.space/spenden-bitcoin-bounties/)

**Comment by Buedenbender** [comment]
Hi, is this still open? It seems like a basic documentation on both topics is touched on in ["Add a new device"](https://github.com/cryptoadvance/specter-desktop/tree/master#add-a-new-device) and ["Create a new wallet"](https://docs.specter.solutions/desktop/#create-a-new-wallet). 

Would you like the current instruction migrated to `USER GUIDE` section, or kept in `INTRODUCTION / Setting up Specter Desktop`? Or keep the current once where they are and just link the new documentation md which are then supposed to appear under `USER GUIDE`

**Comment by k9ert** [comment]
Sorry, i was busy on the pacific Bitcoin conference where we made an announcement using Specter as the exit strategy for Swan Vault. Also, unfortunately we had to focus on Swan Vault the past month and we're really looking forward again to work more on Specter.
I'm not a documentation Expert but i think it make sense to have (as a general guidance) a introduction/tutorial kind of page and in addition a more comprehensive User Guide.  IMHO, connecting an own node should no longer be considered in the introduction but rather in the user guide as the introduction should be for not so technic savy people which should connect via electrum (the future standard way).
But maybe you see it different?!
